### DISCUSSION TITLE
Will `_disableForm` become part of the public API?

### DISCUSSION THREAD

**Discussion by david-crespo** [react-hook-form/react-hook-form#11462] [General]
I noticed `_disableForm` was added in the recent form-level `disabled` work, but it's not documented and has the underscore in front. 

https://github.com/react-hook-form/react-hook-form/blob/6fee03caa768dc612f884f209cf4dea2ba7eb701/reports/api-extractor.md?plain=1#L80

It happens to be used in `useForm` in exactly the way I was hoping to use it in my own code — it a `useEffect` to keep disabled state synced with a prop. (I want to do it this way because I want to do it in a component my `useForm` result is passed to, not at top level in every `useForm` call.)

https://github.com/react-hook-form/react-hook-form/blob/6fee03caa768dc612f884f209cf4dea2ba7eb701/src/useForm.ts#L105-L108

Basically I'm curious how risky it is for me to use this. Is it planned for `_disableForm` to become part of the public API if it is found to work well?
